cibyl/sources/jenkins.py

```diff
@@ -217,6 +221,7 @@ def generate_query_url():
 
             url += f'{api_entrypoint}{query}'
 
+            LOG.debug("URL: %s", url)
```

Comment:
This is probably a debugging log line that we want to remove. If we want to keep it, then I think we should make a bit more explicit (i.e connecting to jenkins with url...)

Reply:
I chose to remove it, because it showed the user personal token in the URL